lib/response.js

```diff
@@ -108,6 +108,9 @@ function generatePayload (response) {
   res.body = res.payload
   res.trailers = response._lightMyRequest.trailers
 
+  // Prepare payload parsers
+  res.json = parseJsonPayload.bind(res)
```

Comment:
I would not bind this, but rather attach it to `res` directly.

Reply:
Done